### ISSUE TITLE
Message 'No handlers could be found for logger "rdflib.term"' clutters output

### ISSUE THREAD

**Issue by jneubert** [RDFLib/pyrdfa3#18]
Running localRDFa.py on Python 2.7.5, I get an invalid output like
```
No handlers could be found for logger "rdflib.term"
@prefix cc: <http://creativecommons.org/ns#> .
@prefix dc: <http://purl.org/dc/elements/1.1/> .
@prefix dcterms: <http://purl.org/dc/terms/> .
```
As a non-pyhton programmer, I'm not sure how I could avoid it. The hint at http://stackoverflow.com/questions/17393664/no-handlers-could-be-found-for-logger-rdflib-term gave an idea, but I'm still not sure what's missing, and how should I add it.

**Comment by iherman** [comment]
Can you give me a file where you hit this issue? Or does it occur with any RDFa content? I am afraid it is an RDFLib issue...

**Comment by iherman** [comment]
Also, do you know what version of RDFLib is used underneath? You can do the following from a terminal:
```
14:19 wd> python
Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 12:39:47)
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import rdflib
INFO:rdflib:RDFLib Version: 4.2.1
>>>
```

**Comment by jneubert** [comment]
The file is generated and a bit lengthy, so I copied it to http://zbw.eu/beta/tmp/jel.html. A check with the markup validation service reveales that I introduced a typo into the template (missing " in a href attribute) with this version. The problem does not occur any more with http://zbw.eu/beta/tmp/jel_fixed.html (or a few other files I've checked).

Sorry for not having validated the output before (felt too save, it's the fifth or so version of this vocabulary).

So it is correct, that an error is thrown, and perhaps also having another error that the initial one could not be logged appropriately. However, it would be nice to have that on stderr instead of stdout, or have a dependency on the logging library during installation.

To answer your second question:
```
Python 2.7.5 (default, Nov  6 2016, 00:28:07)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-11)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import rdflib
>>> print rdflib.__version__
4.1.2
>>>
```
Prior versions of the JEL vocabulary have been converted in a much older environment (Python 2.4.3 with rdflib 2.4.2), where I don't remember similar effects.
